lib/index.js

```diff
@@ -202,7 +202,7 @@ King = new Class({
 	 * @returns {Promise}
      **/
     ,destroy:co.wrap(function* destroy( resource, id ){
-        let resp = yield this.request('delete', resource, id );
+        let resp = yield this.request('delete', resource, id, [200, 404]);
```

Comment:
Still would rather pre-define the acceptable codes for the http methods, rather than having to pass them in everywhere.